collector/collector.go

```diff
@@ -145,6 +146,27 @@ func (c *Collector) collectAllQueues(httpClient *http.Client, result *Result) er
 
 	if c.DebugHttp {
 		trace := &httptrace.ClientTrace{
+			GetConn: func(hostPort string) {
+				fmt.Printf("getting connection to %s: %v\n", hostPort, time.Now())
+			},
+			GotConn: func(info httptrace.GotConnInfo) {
+				fmt.Printf("got connection [reused?: %t, idle?: %t (time: %v)]: %v\n",
+					info.Reused, info.WasIdle, info.IdleTime, time.Now())
+			},
+			PutIdleConn: func(err error) {
+				if err == nil {
+					fmt.Printf("connection successfully put idle: %v\n", time.Now())
+				} else {
+					fmt.Printf("Failed to put connection idle with error - %v: %v\n", err, time.Now())
+				}
+			},
+			GotFirstResponseByte: func() {
+				fmt.Printf("received first response byte: %v\n", time.Now())
+			},
+			Got1xxResponse: func(code int, header textproto.MIMEHeader) error {
+				fmt.Printf("received %d with header %v: %v\n", code, header, time.Now())
+				return err
```

Comment:
`err` here will capture the variable declared on line 132, which has no relation to this closure. 

```suggestion
				return nil
```